src/components/Offer.js

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+
+import { consts, device, texts } from '../config';
+import { matomoTrackingString, momentFormat, trimNewLines } from '../helpers';
+import { useMatomoTrackScreenView } from '../hooks';
+import { GenericType } from '../types';
+import { ImageSection } from './ImageSection';
+import { InfoCard } from './infoCard';
+import { OperatingCompany } from './screens';
+import { StorySection } from './StorySection';
+import { BoldText, RegularText } from './Text';
+import { Title, TitleContainer, TitleShadow } from './Title';
+import { Touchable } from './Touchable';
+import { Wrapper, WrapperRow, WrapperWithOrientation, WrapperWrap } from './Wrapper';
+
+const { MATOMO_TRACKING } = consts;
+
+const isImage = (mediaContent) => mediaContent.contentType === 'image';
+
+// eslint-disable-next-line complexity
+export const Offer = ({ data, navigation }) => {
+  const {
+    companies,
+    contacts,
+    contentBlocks,
+    dataProvider,
+    genericType,
+    mediaContents,
+    payload,
+    publicationDate,
+    sourceUrl,
+    title
+  } = data;
+
+  useMatomoTrackScreenView(
+    matomoTrackingString([
+      genericType === GenericType.Job
+        ? MATOMO_TRACKING.SCREEN_VIEW.JOB_OFFER
+        : MATOMO_TRACKING.SCREEN_VIEW.COMMERCIAL_OFFER,
+      dataProvider && dataProvider.name,
+      title
+    ])
+  );
+
+  const link = sourceUrl?.url;
+  const rootRouteName = navigation.getParam('rootRouteName', '');
+  const headerTitle = navigation.getParam('title', '');
+
+  // action to open source urls
+  const openWebScreen = (webUrl) =>
```

Comment:
this unsexy method seems not to be accurate or not used everywhere. when clicking "der alfigste fred..." links, the browser opens and not our internal web screen.

Reply:
it was missing for the first info card.